### PR TITLE
feat: add notifyUser option to disable the upgrade dialog prompt

### PR DESCRIPTION
**Pull request by mvines**
Related to #22, I want a way to disable the hard coded user prompt when a new update is downloaded.

### CODE REVIEW

**Comment by mvines** [comment]
Sure, done

File: readme.md [hunk 1]
```diff
@@ -57,20 +57,22 @@ Options:
 - `host` String (optional) - Defaults to `https://update.electronjs.org`
 - `updateInterval` String (optional) - How frequently to check for updates. Defaults to `10 minutes`. Minimum allowed interval is `5 minutes`.
 - `logger` Object (optional) - A custom logger object that defines a `log` function. Defaults to `console`. See [electron-log](https://github.com/megahertz/electron-log), a module that aggregates logs from main and renderer processes into a single file.
+- `notifyUser` Boolean (optional) - Defaults to true.  When enabled the user will be
```
Comment (malept) [inline readme.md]:
```suggestion
- `notifyUser` Boolean (optional) - Defaults to `true`.  When enabled the user will be
```

**Comment by mvines** [comment]
Sweet, thanks

**Pull request merged**